### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by imcodingideas**
I fixed some typos as I was digging deep into this starter pack. There shouldn't be any regressions to consider.

## Test Plan

Should have no side effects, all tests should pass.

## Screenshots

n/a

### CODE REVIEW

File: app/root.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ import {
 	DropdownMenuTrigger,
 } from './components/ui/dropdown-menu.tsx'
 import { Icon, href as iconsHref } from './components/ui/icon.tsx'
-import fontStylestylesheetUrl from './styles/font.css'
-import tailwindStylesheetUrl from './styles/tailwind.css'
+import fontStyleStyleSheetUrl from './styles/font.css'
```
Comment (imcodingideas) [inline app/root.tsx:43]: I checked with MDN, and they refer to Stylesheet as StyleSheet. It felt wrong looking at it.

https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet

Comment (kentcdodds) [reply app/root.tsx:43]: Maybe it should be `fontStyleSheetUrl` 😅

Comment (imcodingideas) [reply app/root.tsx]: Yessir, I missed this with renaming haha.

File: app/routes/settings+/profile.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import { json, type DataFunctionArgs } from '@remix-run/node'
 import { Link, Outlet, useMatches } from '@remix-run/react'
+import { z } from 'zod'
```
Comment (imcodingideas) [inline app/routes/settings+/profile.tsx:3]: This was a eslint issue I saw in github

Screenshot 2023-09-18 at 9 07 36 AM

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**